download.md

```diff
@@ -66,7 +66,7 @@ function getVersionList() {
             $('.selectpicker').append('<option value="'+version+'">'+version+'</option>');
           }
           else{
-            $('.selectpicker').append('<option value="'+version+'">'+version+' - legacy, not recommended</option>');
+            $('.selectpicker').append('<option value="'+version+'">'+version+' - legacy, deprecating</option>');
```

Comment:
change to `being deprecated` ?

Reply:
I changed it to add the date: `deprecating on September 27, 2018`